src/ui/server/api/keystone/application/index.ts

```diff
@@ -123,6 +124,38 @@ const application = {
         throw new Error('Updating application policy and export');
       }
     },
+    exporterBroker: async (id: string, update: object) => {
+      try {
+        console.info('Updating application exporter broker');
+
+        const variables = {
+          where: { id },
+          data: update,
+        };
+
+        const response = (await apollo('POST', updateExporterBrokerMutation, variables)) as ApolloResponse;
+
+        if (response.errors) {
+          console.error('GraphQL error updating application exporter broker ', response.errors);
+        }
+
+        if (response?.networkError?.result?.errors) {
+          console.error('GraphQL network error updating application exporter broker ', response.networkError.result.errors);
+        }
+
+        if (response?.data?.updateExporterBroker) {
+          const { data } = response;
+
+          return data.updateExporterBroker;
```

Comment:
`return response.data.updateExporterBroker;`

Just simplified these in previous PR 😊